modules/blocks/process.js

```diff
@@ -476,6 +476,7 @@ __private.receiveForkOne = function (block, lastBlock, cb) {
 	// Keep the oldest block, or if both have same age, keep block with lower id
 	if (block.timestamp > lastBlock.timestamp || (block.timestamp === lastBlock.timestamp && block.id > lastBlock.id)) {
 		library.logger.info('Last block stands');
+		modules.transport.broadcastBlock({broadhash: modules.system.getBroadhash(), unmatchBroadhash: true}, lastBlock);
```

Comment:
Issue https://github.com/LiskHQ/lisk/issues/758 is about improve dealing with forks 5, but change was added to `__private.receiveForkOne` instead of `__private.receiveForkFive`

Reply:
done, thanks